client/src/components/mainContent.jsx

```diff
@@ -90,12 +104,26 @@ const MainContent = () => {
                         {meeting.description}
                       </Typography>
                     </div>
+                    <Divider className={classes.DividerInCardContent}/>
+                    <div className={classes.BelowDividerinCardContent}>
+                      <CardHeader title={meeting.title} />
+                      <Button
+                        variant="outlined"
+                        className={classes.button}
+                      >
+                        CREATE LINK
+                      </Button>
+                    </div>
                   </CardContent>
                 </Card>
               </Grid>
             ))}
           </Grid>
         </Container>
+        {open == true && (
```

Comment:
you can just make it a truthy validation OR do an explicit triple === check or else i think React will probably throw a warning

```suggestion
        {open && (
```

OR

```suggestion
        {open === true && (
```